sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -71,12 +71,11 @@ void Scheduler::waitForRecordToFinish(MemObjRecord *Record,
 EventImplPtr Scheduler::addCG(std::unique_ptr<detail::CG> CommandGroup,
                               QueueImplPtr Queue) {
   EventImplPtr NewEvent = nullptr;
-  const bool IsKernel = CommandGroup->getType() == CG::Kernel;
+  const auto Type = CommandGroup->getType();
```

Comment:
Please use full type name here instead of auto. It is not clear from rhs.

Reply:
done